neuron-explainer/neuron_explainer/explanations/simulator.py

```diff
@@ -713,24 +851,30 @@ async def simulate(
         self,
         tokens: Sequence[str],
     ) -> SequenceSimulation:
-        prompt = self._make_simulation_prompt(
-            tokens,
-            self.explanation,
-        )
-        response = await self.api_client.make_request(
-            prompt=prompt, echo=False, max_tokens=1000
-        )
-        assert len(response["choices"]) == 1
-
-        choice = response["choices"][0]
-        if self.prompt_format == PromptFormat.HARMONY_V4:
+        if self.json_mode:
+            prompt = self._make_simulation_prompt_json(
+                tokens,
+                self.explanation,
+            )
+            response = await self.api_client.make_request(
+                messages=prompt, max_tokens=2000, temperature=0, json_mode=True
+            )
+            assert len(response["choices"]) == 1
+            choice = response["choices"][0]
             completion = choice["message"]["content"]
-        elif self.prompt_format in [PromptFormat.NONE, PromptFormat.INSTRUCTION_FOLLOWING]:
-            completion = choice["text"]
+            predicted_activations = _parse_no_logprobs_completion_json(completion, tokens)
         else:
-            raise ValueError(f"Unhandled prompt format {self.prompt_format}")
-
-        predicted_activations = _parse_no_logprobs_completion(completion, tokens)
+            prompt = self._make_simulation_prompt(
+                tokens,
+                self.explanation,
+            )
+            response = await self.api_client.make_request(
+                messages=prompt, max_tokens=1000, temperature=0
```

Comment:
let's make `max_tokens = 2000` a constant and have the above `make_request` call use it too